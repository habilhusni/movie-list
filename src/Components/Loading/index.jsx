import React from "react";
import { Spin } from "antd";
import "./index.css";

const Loading = () => <Spin className="loading" size="large" />;

export default Loading;
