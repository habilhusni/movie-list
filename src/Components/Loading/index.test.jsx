import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJSON from "enzyme-to-json";

import Loading from "./index.jsx";

Enzyme.configure({ adapter: new Adapter() });

describe("Test Snapshot and Functional", () => {
  let component;

  beforeEach(() => {
    component = shallow(<Loading />);
  });

  it("Testing Loading Snapshot", () => {
    expect(toJSON(component)).toMatchSnapshot();
  });
});
