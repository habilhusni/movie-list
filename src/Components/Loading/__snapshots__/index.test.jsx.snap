// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Snapshot and Functional Testing Loading Snapshot 1`] = `
<Spin
  className="loading"
  size="large"
  spinning={true}
  wrapperClassName=""
/>
`;
